packages/server-wallet/src/wallet/wallet-response.ts

```diff
@@ -86,7 +86,9 @@ export class WalletResponse {
     myIndex: number,
     participants: Participant[]
   ): void {
-    this.createdObjectives.push(objective);
+    // TODO: The objective model currently always sets the participants to an empty array when creating the DBObjective
```

Comment:
Currently `toObjective` always sets the `participants` to an [empty array](https://github.com/statechannels/statechannels/blob/517ed467a5dbc542a89aee837669817a7ccafd3d/packages/server-wallet/src/models/objective.ts#L204).

Reply:
I think we need some consult to @geoknee here. I do not know the reason why `participants` is an empty array. What is the effect of adding `participants` to it?

Reply:
While I think we should figure that out, this change is no longer needed for this PR and will be removed.